bindings/python/htcondor2/_job_event.py

```diff
@@ -9,24 +9,49 @@
 from ._job_event_type import JobEventType
 
 class JobEvent(Mapping):
-
+    """
+    A single event from a job event log.
+
+    A :class:`JobEvent` is a :class:`Mapping` of event properties
+    to event values; the type of the value depends on the property.
+    The names of the properties and their types are currently
+    undocumented.
+
+    Because all events have the ``type``, ``cluster``, ``proc``,
+    and ``timestamp`` properties, these are available as attributes;
+    see below.
+    """
+
+    # It would be nice if this signature didn't appear in the docs,
+    # since instantiating these objects shouldn't be in the API.
+    # ... there's probably a standard Pythonic way ot indicating that.
```

Comment:
Just for future reference/testing `:exclude-members: __init__` may do the job. Not fully sure.

Reply:
Tried it and couldn't get it to work.  There's some code changes we could make to make it harder to try to call the constructor directly, but hopefully this just won't ever become an issue.